backend/app/services/chat/preprocessing/contexts.py

```
@@ -0,0 +1,593 @@
+from sqlalchemy.orm import Session
+
+from app.models.subtask_context import ContextStatus, ContextType, SubtaskContext
+from app.services.context import context_service
+
+logger = logging.getLogger(__name__)
+
+
+async def process_contexts(
+    db: Session,
+    context_ids: List[int],
+    user_id: int,
```

Ruff (0.14.10)

Unused function argument: `user_id`

(ARG001)